microsoft-edge/devtools-guide-chromium/console/console-log.md

```diff
@@ -93,21 +97,22 @@ console.log(w3techs);
 ```
 
 
-### Demo webpage: Logging different variable types
+<!-- ------------------------------ -->
+#### Demo webpage for logging various types of values
 
 To use the `log` function to display different variable types:
 
 <!-- demo 2 of 6 -->
 
 1. Open the demo webpage [Console messages examples: Logging different types](https://microsoftedge.github.io/Demos/devtools-console/logging-types.html) in a new window or tab.
 
-1. Press **Ctrl+Shift+J** (Windows, Linux) or **Command+Option+J** (macOS).  DevTools opens, with the **Console** open in main toolbar.
+1. Press **Ctrl+Shift+J** (Windows, Linux) or **Command+Option+J** (macOS).
 
-   Each type of result is displayed in a different way.
+   DevTools opens, with the **Console** open in the Activity Bar, displaying several types of values.  Each type of result is displayed in a different way.
 
-1. Click the triangles to toggle the information and analyze each result in more detail.
+1. Expand each output entry to analyze each result in more detail.
 
-   Suppose you get many log messages that only display a value, but you don't know where the value originated.  For that case, we added curly braces `{}` around the `x` variable, to provide better grouping.
+   Suppose you get many log messages that only display a value, but you want to display where the value originated from.  For that case, the example code adds curly braces `{}` as a format specifier, around the `x` variable to output the name of the variable, in addition to the value of the variable.
```

Comment:
the middle part of the rewrite is hardest part to understand, while the start & end of the paragraph are the easy to understand takeaway.  might try this resequencing:

The second log message in the example code uses curly braces `{}` around the `x` variable <-- _simple_

to output the name of the variable, in addition to the value of the variable. <-- _simple_

This syntax logs an object with a single property named `x` instead of logging the value of `x` only. <-- _technical_

Reply:
The second log message in the example code uses curly braces `{}` around the `x` variable, to output the name of the variable in addition to the value of the variable.  This syntax logs an object that contains a single property named `x`, instead of logging the value of `x` only.